website/content/docs/nia/terraform-modules.mdx

```diff
@@ -87,6 +99,23 @@ variable "services" {
 }
 ```
 
+### Catalog Services Variable
+
+If you are creating a module with for a [catalog-service condition](/docs/nia/tasks#catalog-services-condition), then you have the option to add the `catalog_services` variable, which contains service registration and tag information. If your module would benefit from consuming this information, you can copy the `catalog_services` variable declaration to your `variables.tf` file in addition to the other variables.
```

Comment:
```suggestion
If you are creating a module for a [catalog-service condition](/docs/nia/tasks#catalog-services-condition), then you have the option to add the `catalog_services` variable, which contains service registration and tag information. If your module would benefit from consuming this information, you can copy the `catalog_services` variable declaration to your `variables.tf` file in addition to the other variables.
```

I think this is what you mean instead of "with for"

Reply:
Good call! Thanks for catching all of these typos and improvements!